packages/random/xorshift.pony

```diff
@@ -10,6 +10,15 @@ class XorShift128Plus is Random
   var _x: U64
   var _y: U64
 
+  new from_64bit(x: U64 = 5489) =>
```

Comment:
Maybe `from_u64` would be more consistent with other APIs?

Same comment applies to other additions.